core/store/src/migrations.rs

```diff
@@ -320,3 +326,126 @@ pub fn migrate_13_to_14(path: &String) {
 
     set_store_version(&store, 14);
 }
+
+pub fn migrate_14_to_15(path: &String) {
```

Comment:
Would be great to add a comment here as well saying that it orders execution outcomes by replaying chunks.